src/components/button/Button.module.scss

```diff
@@ -0,0 +1,374 @@
+@use 'theme';
+@use 'sass:meta';
+
+$buttonSize: var(--button-size);
+$buttonWidth: var(--button-width);
+$disabledOpacity: var(--disabled-opacity);
+$buttonPadding: var(--button-padding);
+$buttonRadius: var(--button-radius);
+$buttonFontWeight: var(--button-font-weight);
+$aminoColor: var(--amino-color, null);
```

Comment:
🤔 I don't think color `null` is valid, so either removing it from the fallback or replace it with something else that's valid.